### DISCUSSION TITLE
🤔🤔🤔

### DISCUSSION THREAD

**Discussion by traffisco** [flareact/flareact#142] [General]
Is the project still maintained? 🤔

**Comment by tj-kev** [comment]
@jplhomer If you don't have time to maintain the project. Would you be willing allow others to help? Merging in the pull requests etc?

I'm sure a number of people would be willing to help out? I'll volunteer as one. @traffisco ?

**Reply by jplhomer** [reply]
Chatted offline with @tj-kev who will assist with maintaining the repo moving forward!

**Comment by jplhomer** [comment]
Wrote out a longer response here: https://github.com/flareact/flareact/discussions/144#discussioncomment-630380
